cvat/apps/documentation/user_guide.md

```diff
@@ -192,37 +250,59 @@ Usage examples:
 
 ### Attribute annotation mode (basics)
 
-- In this mode you can edit attributes with fast navigation between objects and frames using keyboard. Press ``Shift+Enter`` shortcut to enter AAMode. After that it is possible to change attributes using keyboard.
+- In this mode you can edit attributes with fast navigation between objects and frames using keyboard. Press ``Shift+Enter`` shortcut to enter AAMode. After that, you can change attributes using a keyboard.
 
     ![](static/documentation/images/image023.jpg)
 
-- The active attribute will be red. In this case it is ``gender``. Look at the bottom side panel to see all possible shortcuts to change the attribute. Press ``2`` key on your keyboard to assign ``female`` value for the attribute.
+- The active attribute will be red. It is ``gender`` in this case. Look at the bottom side panel to see all possible shortcuts for changing the attribute. Press ``2`` key on your keyboard to assign ``female`` value for the attribute.
 
     ![](static/documentation/images/image024.jpg) ![](static/documentation/images/image025.jpg)
 
-- Press ``Up Arrow``/``Down Arrow`` on your keyboard to go to next/previous attribute. In this case after pressing ``Down Arrow`` you will be able to edit ``Age`` attribute.
+- Press ``Up Arrow``/``Down Arrow`` on your keyboard to go to next/previous attribute. In this case, after pressing ``Down Arrow`` you will be able to edit ``Age`` attribute.
 
     ![](static/documentation/images/image026.jpg) ![](static/documentation/images/image027.jpg)
 
 - Use ``Right Arrow``/``Left Arrow`` keys to move on previous/next image with annotation.
 
 ### Downloading annotations
 
-1. To download latest annotations save all changes first. Press ``Open Menu`` and then ``Save Work`` button. There is ``Ctrl+S`` shortcut to save annotations quickly.
+1. To download the latest annotations, you have to save all changes first. Press ``Open Menu`` and then ``Save Work`` button. There is ``Ctrl+S`` shortcut to save annotations quickly.
 
-2. After that press ``Open Menu`` and then ``Dump Annotation`` button.
+2. After that, press ``Open Menu`` and then ``Dump Annotation`` button.
 
     ![](static/documentation/images/image028.jpg)
 
-3. The annotation will be written into **.xml** file. To find the annotation file go to the directory where your browser saves downloaded files by default. For more information visit [.xml format page](./documentation/xml_format.html).
+3. The annotation will be written into **.xml** file. To find the annotation file, you should go to the directory where your browser saves downloaded files by default. For more information visit [xml format page](/xml_format.html).
 
     ![](static/documentation/images/image029.jpg)
 
+### Task synchronization with a repository
```

Comment:
Need to say that the feature works only if git repository was specified when the task was created.